src/style.css

```
@@ -0,0 +1,57 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-position: center ;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star {
+  background-image: url(./images/star.svg);
+}
+
+.stars--1 .stars__star {
+  background-image: url(./images/star.svg);
+}
+
+.stars--2 .stars__star {
+  background-image: url(./images/star.svg);
+}
+
+.stars--3 .stars__star {
+  background-image: url(./images/star.svg);
+}
+
+.stars--4 .stars__star {
+  background-image: url(./images/star.svg);
+}
+
+.stars--5 .stars__star {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(1) {
```

```suggestion
.stars--1 .stars__star:nth-child(1),
stars--2 .stars__star:nth-child(-n+2),
....
 {
```
You can group selectors with commas, to avoid repeating code